packages/core-js-sdk/src/sdk/validations/index.ts

```diff
@@ -19,7 +25,7 @@ export const withValidations =
   };
 
 export const string = (fieldName: string) => [
```

Comment:
noting that there is only once place that uses `string` which is the webauthn sign in - https://github.com/descope/core-js-sdk/pull/153/files

I prefer to modify this function to also allow `undefined` rather then create 3 types of validations - string, nonempty-string / string-or-undefined

Reply:
What about having an `OR` validator, that will get a list of validators 
and allow us to check if at least one validator is passed?